resources/ts/Components/IntegrationClientCredential.tsx

```diff
@@ -46,30 +45,18 @@ export const IntegrationClientCredentials = ({
   isLive: boolean;
 }) => {
   const { t } = useTranslation();
-  const { config } = usePageProps();
 
-  const clientWithLabels =
-    config.keycloak.enabled || (!isLive && config.keycloak.testClientEnabled)
-      ? [
-          {
-            label: "details.credentials.client_id",
-            value: keycloakClient?.clientId,
-          },
-          {
-            label: "details.credentials.client_secret",
-            value: keycloakClient?.clientSecret,
-          },
-        ]
-      : [
-          {
-            label: "details.credentials.client_id",
-            value: client?.clientId,
-          },
-          {
-            label: "details.credentials.client_secret",
-            value: client?.clientSecret,
-          },
-        ];
+  const clientToShow = keycloakClient && !isLive ? keycloakClient : client;
```

Comment:
It looks correct, but I think we better verify once more in the publiq platform frontend when to show which keys.

@grubolsch You once said the general `KEYCLOAK_ENABLED` variable was supposed to no longer be used?
Only the `KEYCLOAK_TEST_CLIENT_ENABLED`?

How should the frontend behave then on platform-test and on platform (prod)?
Should we change the frontend code to only use `KEYCLOAK_TEST_CLIENT_ENABLED`?
And once production is changed to keycloak with or without `KEYCLOAK_TEST_CLIENT_ENABLED` things should work?

Reply:
yes, KEYCLOAK_ENABLED should be refactored away, we now use a combination of KEYCLOAK_CREATION_ENABLED, KEYCLOAK_TEST_CLIENT_ENABLED.

I am wondering if we cannot make the logic easier?
Just say, "if KEYCLOAK_CREATION_ENABLED is true AND there is a keycloak client, show that one. Otherwise, show the auth0 client".
Then you don't even need KEYCLOAK_TEST_CLIENT_ENABLED.

Reply:
I think your suggestion checks out. If possible I would do it that way

Reply:
Okay will switch the condition to that